atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/table/RegeneratingTable.java

```diff
@@ -95,6 +100,32 @@ public void cleanup() throws Exception {
         this.connector.close();
     }
 
+    @State(Scope.Benchmark)
+    public static class VersionedCellRegeneratingTable extends RegeneratingTable<Cell> {
+        private static final Cell cell = Cell.create(PtBytes.toBytes("r"), PtBytes.toBytes("c"));
+        private static final byte[] value = PtBytes.toBytes("v");
+
+        private static final Multimap<Cell, Value> allVersions = getVersions();
+
+        @Override
+        public void setupTableData() {
+            getKvs().truncateTable(getTableRef());
+            getKvs().putWithTimestamps(getTableRef(), allVersions);
+        }
+
+        @Override
+        public Cell getTableCells() {
+            return cell;
+        }
+
+        private static Multimap<Cell, Value> getVersions() {
+            Multimap<Cell, Value> versions = ArrayListMultimap.create();
+            LongStream.range(1, CELL_VERSIONS + 1).boxed()
```

Comment:
nit: `LongStream.rangeClosed(1, CELL_VERSIONS)` might be a little bit clearer

Reply:
👍